pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1876,6 +1876,59 @@ def drop_table(self, table_name):
         """
         self.add_remove_table(table_name, False)
 
+    def list_tables(self):
+        """
+        Returns a list with the names of all created tables.
+
+        Example
+        --------
+
+        >>> from blazingsql import BlazingContext
+        >>> bc = BlazingContext()
+        >>> bc.create_table('product_reviews', "product_reviews/*.parquet")
+        >>> bc.create_table('store_sales', "store_sales/*.parquet")
+        >>> bc.create_table('nation', "nation/*.parquet")
+        >>> tables = bc.list_tables()
+        >>> print(tables)
+                  ['product_reviews', 'store_sales', 'nation']
+        """
+        return list(self.tables.keys())
+
+    def describe_table(self, table_name):
+        """
+        Returns a dictionary with the names of all the columns and their types
+        for the specified table. A ValueError is thrown if thr table is not found.
+
+        Parameters
+        ----------
+
+        table_name : string of the table name to describe
+
+        Example
+        --------
+
+        >>> from blazingsql import BlazingContext
+        >>> bc = BlazingContext()
+        >>> bc.create_table('nation', "nation/*.parquet")
+        >>> info_table = bc.describe_table("nation")
+        >>> print(info_table)
+                  {'n_nationkey': 'int32', 'n_name': 'object',
+                   'n_regionkey': 'int32', 'n_comment': 'object'}
```

Comment:
is the type shown for a string column `object`? i understand why, but i think we want it to display `str`

Reply:
have a funtion which maps the data types to user friendly strings to display to the user

Reply:
Updating:   `object`   to `str`